.github/scripts/trymerge.py

```diff
@@ -828,7 +828,7 @@ def has_internal_changes(self) -> bool:
             return False
         return checks[checkrun_name][0] != "SUCCESS"
 
-    def merge_ghstack_into(self, repo: GitRepo, force: bool, comment_id: Optional[int] = None) -> None:
+    def merge_ghstack_into(self, repo: GitRepo, force: bool, comment_id: Optional[int] = None, land_check_commit: str = None) -> None:
```

Comment:
should we make the land_check_commit an optional str?

Reply:
Yup, definitely